test/bench_test/CMakeLists.txt

```diff
@@ -12,11 +12,11 @@ if(NOT Boost_UNIT_TEST_FRAMEWORK_FOUND)
     cm_find_package(Boost REQUIRED COMPONENTS unit_test_framework)
 endif()
 
-add_custom_target(algebra_runtime_bench_tests)
+add_custom_target(${CURRENT_PROJECT_NAME}_bench)
 
-macro(define_runtime_algebra_test name)
-    set(test_name "algebra_${name}_bench_test")
-    add_dependencies(algebra_runtime_bench_tests ${test_name})
+macro(define_bench name)
+    set(test_name "algebra_${name}_bench")
```

Comment:
As for me, it's better to name it with `_test` suffix, but never mind

Reply:
Those tests shall not be executed in the normal workflow